### PR TITLE
[DCOS-2409] Improved error message for `dcos task log <completed_task>`

### PR DESCRIPTION
**Pull request by mgummelt**

### CODE REVIEW

File: cli/dcoscli/task/main.py [hunk 1]
```diff
@@ -153,6 +153,18 @@ def _log(follow, completed, lines, task, file_):
 
     mesos_files = _mesos_files(completed, fltr, file_)
     if not mesos_files:
+        if not completed:
+            completed_files = _mesos_files(True, fltr, file_)
+            if completed_files:
+                msg = 'No running tasks match ID [{}], however there '.format(
```
Comment (tamarrow) [inline cli/dcoscli/task/main.py]: Should the punctuation be `... ID [{}]; however, there ...`

**Comment by tamarrow** [comment]
one minor comment, and then LGTM once tests pass.

**Comment by mgummelt** [comment]
Master is failing, and I think #297 fixes it, so this is blocked on #297

**Comment by tamarrow** [comment]
@mgummelt #297 is now merged

File: cli/dcoscli/task/main.py (hunk 1 above)
Comment (ssk2) [inline cli/dcoscli/task/main.py]: Agreed!

Comment (mgummelt) [inline cli/dcoscli/task/main.py]: fixed

**Pull request merged**